CMakeLists.txt

```diff
@@ -319,44 +321,25 @@ ELSE()
 ENDIF()
 
 IF(NOT OPENGL_BACKEND STREQUAL "None")
-	SET(OPENGL_BACKEND 1)
+
 	IF(SDL_BACKEND STREQUAL "SDL")
-		MESSAGE(FATAL_ERROR "SDL2 is required for the OpenGL custom code!")
+		MESSAGE(FATAL_ERROR "SDL2 is required for OpenGL backend support!")
 	ENDIF()
 
-	IF(RPI)
- 		MESSAGE(STATUS "RPI_GLES_PATH: ${RPI_GLES_PATH}")
- 		MESSAGE(STATUS "RPI: ${RPI}")
- 		MESSAGE(STATUS "Setting _OPENGL_INCLUDE_PATH: ${RPI_GLES_PATH}/include")
- 		MESSAGE(STATUS "Setting _OPENGL_LIB_PATH: ${RPI_GLES_PATH}/libs")
- 		SET(_OPENGL_INCLUDE_PATH ${RPI_GLES_PATH}/include)
- 		SET(_OPENGL_LIB_PATH ${RPI_GLES_PATH}/lib)
- 		# for newer cmake
- 		MESSAGE(STATUS "Checking for ${RPI_GLES_PATH}/GLES2/GL2.h and ${RPI_GLES_PATH}/lib")
- 		IF(EXISTS ${RPI_GLES_PATH}/include/GLES2/GL2.h AND EXISTS ${RPI_GLES_PATH}/lib)
- 			MESSAGE(STATUS "Found ${RPI_GLES_PATH}/include/GLES2/GL2.h and ${RPI_GLES_PATH}/lib.")
- 			MESSAGE(STATUS "Setting OPENGL_EGL_INCLUDE_DIRS ${RPI_GLES_PATH}/include/GLES2/GL2.h")
- 			MESSAGE(STATUS "Setting OPENGL_LIBRARIES ${RPI_GLES_PATH}/lib")
- 			SET(OpenGL_EGL_FOUND "True")
- 			SET(OPENGL_EGL_INCLUDE_DIRS RPI_${RPI_GLES_PATH}/include)
- 			SET(OPENGL_LIBRARIES ${RPI_GLES_PATH}/lib)
- 		ENDIF()
- 	ENDIF()
-
-	INCLUDE(FindOpenGL)
-	IF(NOT OPENGL_FOUND)
-		MESSAGE(FATAL_ERROR "OpenGL library not found!")
+	IF(OPENGL_BACKEND STREQUAL "OpenGL")
+		INCLUDE(FindOpenGL)
+		IF(NOT OPENGL_FOUND)
+			MESSAGE(FATAL_ERROR "OpenGL library not found!")
+		ENDIF()
 	ENDIF()
 
-	IF(NOT APPLE)
-		INCLUDE(FindGLEW)
-		IF(GLEW_FOUND)
-			MESSAGE(STATUS "Looking for Glew: found")
-		ELSE()
-			MESSAGE(SEND_ERROR "Looking for Glew: not found!")
-			MESSAGE(FATAL_ERROR "Please install the Glew library and headers first!")
+	IF(OPENGL_BACKEND STREQUAL "GLES")
```

Comment:
Nit: this could be an ELSEIF conjoined to the previous block to save one comparison in one case.

Reply:
Done.